cmd/server/app/migrate_up.go

```diff
@@ -127,10 +130,107 @@ var upCmd = &cobra.Command{
 			return fmt.Errorf("error preparing authz client: %w", err)
 		}
 
-		return nil
+		store := db.NewStore(dbConn)
+		return migrateProjectsRootAndMetadata(ctx, cmd, store)
 	},
 }
 
+func migrateProjectsRootAndMetadata(ctx context.Context, cmd *cobra.Command, store db.Store) error {
+	cmd.Println("Migrating projects to have no parent...")
+	projs, err := store.ListNonOrgProjects(ctx)
+	if err != nil {
+		return fmt.Errorf("error while listing non-org projects: %w", err)
+	}
+
+	var lastError error
+	for _, p := range projs {
+		if err := migrateOneProject(ctx, cmd, store, p); err != nil {
+			cmd.Printf("Error while migrating project %s: %v\n", p.ID.String(), err)
+			lastError = err
+		}
+	}
+
+	if lastError != nil {
+		return lastError
+	}
+
+	// Nothing was done with the old org projects, so we can just delete them
+	cmd.Println("Deleting old org projects...")
+	projs, err = store.ListOldOrgProjects(ctx)
+	if err != nil {
+		return fmt.Errorf("error while listing old org projects: %w", err)
+	}
+
+	for _, p := range projs {
+		if err := deleteOrgProject(ctx, store, p); err != nil {
+			cmd.Printf("Error while deleting project %s: %v\n", p.ID.String(), err)
+			lastError = err
+		}
+	}
+
+	if lastError != nil {
+		return lastError
+	}
+
+	return nil
+}
+
+func migrateOneProject(ctx context.Context, cmd *cobra.Command, store db.Store, p db.Project) error {
+	m := projects.NewSelfEnrolledMetadata()
+	if err := json.Unmarshal(p.Metadata, &m); err != nil {
+		return fmt.Errorf("error while marshalling old metadata: %w", err)
+	}
+
+	newmeta, err := json.Marshal(&m)
+	if err != nil {
+		return fmt.Errorf("error while marshalling new metadata: %w", err)
+	}
+
+	if p.ParentID.Valid {
```

Comment:
Doesn't this still break parent-child projects?  We don't have any yet, but I'd rather find all the org projects and cut their descendants than find all child projects and give them no parents.

(I know that we think we're going to remove this code before we have child projects, but...)

Reply:
ASCII art of what I mean:

```
  [[org]]
  /    \
[A]    [B]
 |
[C]
```

Becomes:
```
[A] [C] [B]
```